pkg/uefi/Dockerfile

```diff
@@ -30,7 +30,7 @@ COPY rpi /rpi
 FROM build AS build-amd64-versions
 
 ENV EDK_VERSION edk2-stable202408
```

Comment:
`edk2-stable202408.01` ?

Reply:
just a comment above it instead of renaming?

Reply:
sure, I'm lazy.